### DISCUSSION TITLE
第一次创建表提示容量不足

### DISCUSSION THREAD

**Discussion by yclxiao** [apache/doris#14073] [A - General / Q&A]
我的磁盘是50G，第一次创建表提示容量不足。

errCode = 2, detailMessage = Cluster default_cluster has no available capacity

CREATE TABLE IF NOT EXISTS demo.expamle_tbl
(
    `user_id` LARGEINT NOT NULL COMMENT "用户id",
    `date` DATE NOT NULL COMMENT "数据灌入日期时间",
    `city` VARCHAR(20) COMMENT "用户所在城市",
    `age` SMALLINT COMMENT "用户年龄",
    `sex` TINYINT COMMENT "用户性别",
    `last_visit_date` DATETIME REPLACE DEFAULT "1970-01-01 00:00:00" COMMENT "用户最后一次访问时间",
    `cost` BIGINT SUM DEFAULT "0" COMMENT "用户总消费",
    `max_dwell_time` INT MAX DEFAULT "0" COMMENT "用户最大停留时间",
    `min_dwell_time` INT MIN DEFAULT "99999" COMMENT "用户最小停留时间"
)
AGGREGATE KEY(`user_id`, `date`, `city`, `age`, `sex`)
DISTRIBUTED BY HASH(`user_id`) BUCKETS 1
PROPERTIES (
    "replication_allocation" = "tag.location.default: 1"
);

**Comment by LemonLiTree** [comment]
show backends\G 看下AvailCapacity是否正常？

**Comment by TaoZex** [comment]
Do you solve this problem？I have the same problem.

**Reply by dataalive** [reply]
most likely is the backends not normal or have not been added to the cluster
